src/main/scala/mesosphere/marathon/storage/StorageConf.scala

```diff
@@ -28,6 +28,12 @@ trait StorageConf extends ZookeeperConf with BackupConf {
     default = Some(50)
   )
 
+  lazy val zkCleaningInterval = opt[Int](
```

Comment:
The name is misleading. I'd prefer something pointing to the GcActor e.g. `gcActorScanInterval`